testing/model-mesh/5-dump-and-clear-prom-db.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+./run_toolbox.py cluster capture_environment
+./run_toolbox.py cluster reset_prometheus_db
```

Comment:
need `dump_prometheus_db` here

Reply:
Solved, I think, thx

Reply:
can you replace this line (which is useless at the moment) by 
```
./run_toolbox.py rhods dump_prometheus_db
```
and add in the publish file:
```
mv $ARTIFACT_DIR/*__rhods__dump_prometheus_db/prometheus.tgz $results_dir/artifacts-sutest/prometheus_rhods.tgz
```